packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -101,6 +100,71 @@ enum HeroFlightDirection {
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=Be9UH1kXFDw}
 ///
+/// {@tool dartpad --template=stateful_widget_scaffold_center}
+/// This sample shows a [Hero] used within a [ListTile]. Tapping on the Hero-wrapped rectangle triggers a hero
+/// animation as a new [MaterialPageRoute] is pushed. Both the size and location of the rectangle animates.
+///
+/// Both widgets use the same [Hero.tag]. The Hero widget uses the matching tags to identify and execute this
+/// animation.
+///
+/// ```dart
+///  Widget build(BuildContext context) {
+///    return buildDemoWidget(context);
+///  }
+/// ```
+///
+/// ```dart preamble
+///  Widget _blueRectangle(Size size) {
+///   return Container(
+///     width: size.width,
+///     height: size.height,
+///     color: Colors.blue,
+///    );
+///  }
+///
+///  Widget buildDemoWidget(BuildContext context) {
+///    return Column(
+///      crossAxisAlignment: CrossAxisAlignment.start,
+///      children: <Widget>[
+///        SizedBox(
+///          height: 20.0,
+///        ),
+///        ListTile(
+///          leading: Hero(
+///              tag: 'hero-rectangle',
+///              child: _blueRectangle(Size(50,50)),
+///            ),
+///            onTap: () => _gotoDetailsPage(context),
+///          ),
+///          title: Text('Tap on the icon to view hero animation transition.'),
```

Comment:
formatting and parentheses:

```suggestion
///        ListTile(
///          leading: Hero(
///            tag: 'hero-rectangle',
///            child: _blueRectangle(Size(50,50)),
///          ),
///          onTap: () => _gotoDetailsPage(context),
///          title: Text('Tap on the icon to view hero animation transition.'),
///        ),
```

Reply:
Done Thanks